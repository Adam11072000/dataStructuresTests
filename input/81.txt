init 8312
deleteImage 2085
getHighestScoredLabel 3813 -7
quit