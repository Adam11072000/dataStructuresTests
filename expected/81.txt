init done.
deleteImage: FAILURE
getHighestScoredLabel: INVALID_INPUT
quit done.
